advent_of_code_day1/ruby.rb

```diff
@@ -0,0 +1,25 @@
+# "test"
+
```

Comment:
remove #test and clean up these notes - include an explanation at the top for what this problem is - include a link to advent-of-code day 1